docs/data-sources/maintenance_window.md

```diff
@@ -0,0 +1,54 @@
+---
+layout: "wavefront"
+page_title: "Wavefront: Maintenance Window"
+description: |-
+    Get the information about a specific Wavefront maintenance window.
+---
+
+# Data Source: wavefront_maintenance_window
+
+Use this data source to get information about a Wavefront maintenance window by its ID.
+
+## Argument Reference
+* `id` - (Required) The ID of the maintenance window.
+
+## Example Usage
+
+```hcl
+# Get the information about specific maintenance window.
+data "wavefront_maintenance_window" "example" {
+  id = "sample-maintenance-window-id"
+}
+```
+
+## Attribute Reference
+
+* `id` -  The ID of the maintenance window.
+* `event_name` - The event name of the maintenance window.
+* `reason` - The reason for the maintenance window.
+* `title` - The title of the maintenance window.
+* `customer_id` - The ID of the customer in Wavefront.
+* `running_state` - The running state of the maintenance window.
+* `start_time_in_seconds` - The start time in seconds after 1 Jan 1970 GMT.
+* `end_time_in_seconds` - The end time in seconds after 1 Jan 1970 GMT.
+* `relevant_customer_tags` - The list of alert tags whose matching alerts will be put into maintenance because
+  of this maintenance window. At least one of `relevant_customer_tags`, `relevant_host_tags`, or `relevant_host_names`
+  is required.
+* `relevant_host_tags` - The list of source/host tags whose matching sources/hosts will be put into maintenance
```

Comment:
The list of source or host tags whose matching sources or hosts will be put into maintenance